### DISCUSSION TITLE
Speech to Text Capability

### DISCUSSION THREAD

**Discussion by spamenigma** [skyeBreach/ueepathfinders-helpdesk#3] [Ideas]
### Describe the feature you'd like to request

STT: The voice output of someone to be transcribed into a (doc or scrolling window). Showing who it is, channel/net spoken on, transcribe what was said, timestamped... (from a perspective of what the user using this is permitted to hear ofc!)

### Describe the solution you'd like to see

.

**Comment by skyeBreach** [comment]
Potentially could utilize [WhisperDock](https://github.com/ErcinDedeoglu/WhisperDock) to implement this feature
